tasks/update-granules-cmr-metadata-file-links/tests/test-index.js

```diff
@@ -122,7 +122,7 @@ test.serial('Should update existing etag on CMR metadata file', async (t) => {
   t.not(newEtag, previousEtag);
 });
 
-test.serial('update-granules-cmr-metadata-file-links throws an error when cmr file type is distribution and no distribution endpoint is set', async (t) => {
+test.serial('update-granules-cmr-metadata-file-links throws an error when cmr file type is distribution/both and no distribution endpoint is set', async (t) => {
```

Comment:
this should probably be just type is both, since that's what's being tested.
```suggestion
test.serial('update-granules-cmr-metadata-file-links throws an error when cmr file type is both and no distribution endpoint is set', async (t) => {
```